tests/integration/simple_integration_test.go

```diff
@@ -0,0 +1,164 @@
+
+			// We could use also kubectl wait --for=condition=Ready pod -l name=cass-operator
+
+			// Deploy traefik
+			// helm install traefik traefik/traefik -n traefik --create-namespace -f docs/content/en/docs/topics/ingress/traefik/kind-deployment/traefik.values.yaml
+			//valuesPath, _ := filepath.Abs("../../docs/content/en/docs/topics/ingress/traefik/kind-deployment/traefik.values.yaml")
+			//helm.RunHelmCommandAndGetOutputE(GinkgoT(), options, "install", "traefik", "traefik/traefik", "-n", "traefik", "--create-namespace", "-f", valuesPath)
+
+			// Deploy the cluster
+			clusterChartPath, err := filepath.Abs("../../charts/k8ssandra-cluster")
+			Expect(err).To(BeNil())
+
+			releaseNameCluster = releaseName + "-cluster"
+			options = &helm.Options{
+				// Enable traefik to allow redirections for testing
+				SetValues: map[string]string{
+					"ingress.traefik.enabled": "true",
+					"ingress.traefik.monitoring.grafana.host": "grafana.localhost",
+					"ingress.traefik.monitoring.prometheus.host": "prometheus.localhost",
+					"ingress.traefik.repair.host": "repair.localhost",
+				},
+				KubectlOptions: k8s.NewKubectlOptions("", "", namespace),
+			}
+			helm.Install(GinkgoT(), options, clusterChartPath, releaseNameCluster)
+
+			// Wait for CassandraDatacenter to be ready..
+			k8s.RunKubectl(GinkgoT(), kubectlOptions, "wait", "--for=condition=Ready", "cassandradatacenter/dc1", "--timeout=300s")
+
+			// Verify all the correct services are there
+			// Grafana
+			service := k8s.GetService(GinkgoT(), kubectlOptions, "grafana-service")
+			Expect(service).ToNot(BeNil())
+
+			// Prometheus
+			service = k8s.GetService(GinkgoT(), kubectlOptions, "prometheus-operated")
+			Expect(service).ToNot(BeNil())
+
+			// Cassandra
+			service = k8s.GetService(GinkgoT(), kubectlOptions, "k8ssandra-dc1-service")
+			Expect(service).ToNot(BeNil())
+			service = k8s.GetService(GinkgoT(), kubectlOptions, "k8ssandra-dc1-all-pods-service")
+			Expect(service).ToNot(BeNil())
+			service = k8s.GetService(GinkgoT(), kubectlOptions, "k8ssandra-seed-service")
+			Expect(service).ToNot(BeNil())
+
+			// Reaper
+			// Medusa (if enabled)
+
+			// Verify traefik is ready
+			kubectlOptions = k8s.NewKubectlOptions("", "", "traefik")
+			k8s.RunKubectl(GinkgoT(), kubectlOptions, "wait", "--for=condition=Ready", "pod", "-l", "app.kubernetes.io/name=traefik", "--timeout=300s")
+			kubectlOptions = k8s.NewKubectlOptions("", "", namespace)
+
+			// TODO Verify with cqlsh that Cassandra is working properly
+
+			// Verify that prometheus is polling the Cassandra instance correctly
+			// Wait for the Prometheus to be ready
+			k8s.RunKubectl(GinkgoT(), kubectlOptions, "wait", "--for=condition=Ready", "pod", "-l", "app=prometheus", "--timeout=300s")
+
+			// Poll Prometheus targets and check that it contains our cluster
+			res, err := http.Get("http://prometheus.localhost:8080/api/v1/targets")
```

Comment:
Should this part be commented out for now?

Reply:
..why?

Reply:
The test failed for me the first time with the Prometheus checks. I assumed there was an issue with ingress. I didn't realize that I was running against a GKE cluster. I just ran again against a kind cluster configured with `./tests/bin/cluster.sh` and everything passed 👌